pkg/testutil/testutil_linux.go

```diff
@@ -40,7 +40,7 @@ var (
 	WordpressIndexHTMLSnippet   = "<title>WordPress &rsaquo; Installation</title>"
 	MariaDBImage                = mirrorOf("mariadb:10.5")
 	DockerAuthImage             = mirrorOf("cesanta/docker_auth:1.7")
-	FluentdImage                = mirrorOf("fluent/fluentd:v1.14-1")
+	FluentdImage                = "fluent/fluentd:v1.17.0-debian-1.0"
```

Comment:
I assume we generally avoid Docker Hub because of the dreaded 429? (which I guess is why we mirror stuff on ghcr?)
I also think this approach is problematic (as obviously publishing there is a process we cannot control from this repo).
I would prefer a solution where we cache the images on the CI.
Also, we might want to start pulling by digest for these.

Anyhow, this is fine ^ - and these problems can be addressed later.